lib/rails-jquery-autocomplete/autocomplete.rb

```diff
@@ -98,7 +98,7 @@ def get_object(model_sym)
     #
     def json_for_autocomplete(items, method, extra_data=[])
       items = items.collect do |item|
-        hash = {"id" => item.id.to_s, "label" => item.send(method), "value" => item.send(method)}
+        hash = HashWithIndifferentAccess.new({"id" => item.id.to_s, "label" => item.send(method), "value" => item.send(method)})
```

Comment:
Line is too long. [128/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.